ui/desktop/app/routes/scopes/scope/projects.js

```diff
@@ -49,4 +54,46 @@ export default class ScopesScopeProjectsRoute extends Route {
     );
     return projects;
   }
+
+  /**
+   * Poll for new sessions with credentials. Sends a notification for each new session that has a credential.
+   */
+  @runEvery(2000)
+  async poller() {
+    const sessions =
+      await this.clientAgentSessions.getNewSessionsWithCredentials();
+
+    sessions.forEach((session) => {
+      new window.Notification(
+        this.intl.t('notifications.connected-to-target.title', {
+          target: session.alias,
+        }),
+        {
+          body: this.intl.t('notifications.connected-to-target.description'),
+        },
+      ).onclick = async () => {
+        let orgScope = 'global';
+
+        try {
+          const aliases = await this.store.query('alias', {
+            recursive: true,
```

Comment:
do we need this recursive, since aliases are only available at the global level?

Reply:
Also, should we query list-resolvable aliases or this? When would we use the other endpoint?

Reply:
Yeah recursive isn't needed here, I'll update that. The trick is this is actually querying for resolvable aliases haha, see [here](https://github.com/hashicorp/boundary-ui/blob/79a932bebb57f9759589bf1384f7e7dc8356fbe5/addons/api/addon/handlers/client-daemon-handler.js#L30)